torch_xla/csrc/runtime/stablehlo_composite_helper.cc

```diff
@@ -277,6 +278,11 @@ class BuildStableHLOCompositePass : public mlir::OperationPass<mlir::ModuleOp> {
       }
     }
 
+    if (!mlir::sortTopologically(composite_op->getBlock())) {
+      composite_op->emitError()
+          << "The graph is not acyclic after the composite creation pass.";
```

Comment:
The graph is not acyclic after BuildStableHLOCompositePass

Reply:
Yeah, just would like to catch the error if `mlir::sortTopologically` failed.

Reply:
I mean changing the message from "after the composite creation pass" to "after BuildStableHLOCompositePass" to be more precise lol.

Reply:
sg, updated